tutorials/gm-world.md

```diff
@@ -6,7 +6,7 @@ description: Build a sovereign rollup with Ignite CLI, Celestia, and Rollkit loc
 
 ## 🌞Introduction {#introduction}
 
-This tutorial will guide you through building a sovereign `gm-world` rollup (`gm` stands for "good morning") using Rollkit. Unlike the [Quick Start Guide](https://rollkit.dev/tutorials/quick-start), this tutorial provides a more practical approach to understanding sovereign rollup development.
+This tutorial will guide you through building a sovereign `gm-world` rollup (`gm` stands for "good morning") using Rollkit. Unlike the [Quick start guide](https://rollkit.dev/tutorials/quick-start), this tutorial provides a more practical approach to understanding sovereign rollup development.
```

Comment:
should Quick also be lowercase?

Reply:
i was wondering this too...honestly not sure but probably?